### PR TITLE
New: Add Brazilian TLD support

### PR DESCRIPTION
**Pull request by willianveiga**
References:

- https://en.wikipedia.org/wiki/.br
- https://registro.br/dominio/categoria.html

### CODE REVIEW

File: spec/email_inquire/inquirer_spec.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@
 describe EmailInquire::Inquirer do
   context "considering constants" do
     domain_regexp = /\A[a-z0-9.-]+[a-z0-9]\.[a-z]+\z/
-    tld_regexp = /\A\.[a-z0-9.]+[a-z0-9]\.[a-z]+\z/
+    tld_regexp = /\A\.[a-z0-9]+\.[a-z]+\z/
```
Comment (maximeg) [inline spec/email_inquire/inquirer_spec.rb]: That's not to have "foo..com" matching.

**Comment by maximeg** [comment]
Thanks for your contribution.

I refactored a little bit to prepare your `.com.br` landing.
You'll have to merge master and adapt.

**Comment by willianveiga** [comment]
Thank you very much for your code review, Maxime.

Adaptation done with your changes. I hope it's ok now.

I'm looking forward to your feedback 😄

**Comment by maximeg** [comment]
Released with v0.6.0

**Pull request merged**